indico/web/client/styles/partials/_object-lists.scss

```diff
@@ -126,9 +126,13 @@
 }
 
 .list-filter {
-  padding-bottom: 14em;
   position: relative;
 
+  .list-filter-content {
+    max-height: 45em;
```

Comment:
do you really want `em` here and not `vh`?

Reply:
the size looks fine btw, just wondered if it wouldn't be better to make it relative to the viewport height

Reply:
It makes sense to use `vh`.